content/docs/user-guide/data-pipelines/index.md

```diff
@@ -0,0 +1,14 @@
+# Data Pipelines
+
+In machine learning, _data pipelines_ start with data inputs, then process them
```

Comment:
let's skip / remove this paragraph - no time fixing it, and again too many questions

Reply:
Removed in f9f0a59c3.